frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -178,6 +178,11 @@ const plugin: Plugin<ConsumedExtensions> = [
             throw e;
           });
       },
+      accessReview: {
```

Comment:
This should be a function that takes in the obj and kind and returns the access review object.